requirements/base.txt

```diff
@@ -3,20 +3,19 @@
 -r wheels.txt
 
 # edX Internal Requirements
-git+https://github.com/edx/XBlock.git@9c634481dfc85a17dcb3351ca232d7098a38e10e#egg=XBlock
+git+https://github.com/edx/XBlock.git@django1.8-upgrade#egg=XBlock
```

Comment:
@macdiesel Currently XBlock is handled as per https://openedx.atlassian.net/wiki/display/TNL/edX+libs+Django+upgrade+1.4+to+1.8
So we can add tag here once XBlock is ready